app/views/early_career_payments/claims/itt_year.html.erb

```diff
@@ -33,8 +33,7 @@
             <% end %>
 
             <div class="govuk-radios">
-<!--              TODO this needs to get the year from the PolicyConfiguration-->
-              <% JourneySubjectEligibilityChecker.selectable_itt_years_for_claim_year(AcademicYear.new(2022)).each do |year| %>
+              <% JourneySubjectEligibilityChecker.selectable_itt_years_for_claim_year(AcademicYear.for(Date.today)).each do |year| %>
```

Comment:
Same comment as above: I think we need to get the academic year for the current `PolicyConfiguration`, rather than the current calendar academic year. E.g. `PolicyConfiguration.for(LevellingUpPremiumPayments).current_academic_year`